nipype/pipeline/engine/base.py

```diff
@@ -18,18 +18,14 @@
                         absolute_import)
 from builtins import object
 
-from future import standard_library
-standard_library.install_aliases()
```

Comment:
I meant not to bother removing or reordering, and just tolerate the PEP8 violation. Unless it's causing problems, let's not change this here.

Reply:
I've checked locally with Python 2 and this aliases are not necessary. Let's get rid of them whenever possible.

Reply:
@oesteban @effigies I'm not sure if I understand, does `standard_library.install_aliases()`  at the end of the import block (e.g. in node.py or workflow.py) changes anything?

Reply:
I don't believe it does.